File: RAI/Clase15/Python/InnerClasses/Main.py
from Herramientas import Herramientas

if __name__ == "__main__":

    caja = Herramientas("mi caja")
    print(caja.martillo)

    mi_martillo = Herramientas.Martillo(16, "madera", "oro")
    print(mi_martillo)
    

File: RAI/Clase15/Java/InnerClasses/Main.java
//package RAI.Clase15.Java.InnerClasses;

public class Main {
    public static void main(String[] args) {        
        Herramientas caja = new Herramientas("mi maleta");
        System.out.println(caja.getMartillo());

        Herramientas.Martillo niMartillo = caja.crearMartillo(16, "madera", "oro");
        System.out.println(niMartillo);

    }
}
